### ISSUE TITLE
Issue with k8s.io/docs/tutorials/hello-minikube/

### ISSUE THREAD

**Issue by linuxman1000** [kubernetes/website#21254]
Click to view port on host1:  502 Bad Gateway.  Everytime.

**Comment by RA489** [comment]
@linuxman1000 I don’t yet have a clear way to replicate this issue. Do you mind adding some additional details. Here is additional information that would be helpful:

* The exact minikube start command line used

* The full output of the minikube start command, preferably with --alsologtostderr -v=4 for extra logging.

* The full output of minikube logs

* The full output of kubectl get po -A

Thank you for sharing your experience!

**Comment by linuxman1000** [comment]
I'm attempting to gather more information on this.  Firstly, I need to replicate the issue.  If I'm able, I'll then record the steps taken to do so and I'll post that information here.

**Comment by linuxman1000** [comment]
Update:  I've attempted to replicate the 502 Bad Gateway response that I originally received.  However, it seems that everything is working now.  When I follow the instructions at https://kubernetes.io/docs/tutorials/hello-minikube/, I no longer have an issue.  I'm able to see another browser tab with Katacoda prompting for the port number.  I typed in 30000 and I'm able to see the Kubernetes dashboard (which, I think is running from my local machine, port 30000).
